lib/utilities/debate_events/src/lincoln_douglas.dart

```diff
@@ -26,4 +26,12 @@ class LincolnDouglas extends DebateEvent {
       useAffNeg: true,
     );
   }
+
+  @override
+  int get hashCode => 1;
+
+  @override
+  bool operator ==(Object other) {
+    return this.hashCode == other.hashCode;
+  }
```

Comment:
Not sure if this is correct.

I want the result of this code bit to be that two `LincolnDouglas` debate events are always identical. I think the right way to do this is with a const constructor but you can't have a const constructor if you call methods in the constructor body because those methods are --by definition-- not const. So the other way to get this behavior is to override both the hashCode and the == operator, but that kinda feels hacky.

What do you think @sorenrood? Can I get a code review here :)

Reply:
I'm also gonna cc you on the same problem in another file.

Reply:
I don't know why my github app didn't notify me about this. Huh.